ambariclient/base.py

```diff
@@ -396,7 +397,11 @@ class Model(object):
     min_version = OLDEST_SUPPORTED_VERSION
 
     def __init__(self, parent, data=None):
-        self._data = data if data is not None else {}
+        if data is None:
+            data = {}
+
+        self._data = dict((key, value) for key, value in six.iteritems(data)
```

Comment:
No, it's probably a good change.  I'll pull master into my dev environment after this is merged and make sure we don't have any regressions in our codebase.